src/storage/src/hummock/store/local_hummock_storage.rs

```diff
@@ -350,9 +350,7 @@ impl LocalStateStore for LocalHummockStorage {
     }
 
     async fn try_flush(&mut self) -> StorageResult<()> {
-        if self.mem_table_spill_threshold != 0
```

Comment:
I think we should keep using `0` to disable memtable spill. Otherwise, for those who has explicitly set `mem_table_spill_threshold` to 0 will see excessive spill in their cluster after this PR.

Reply:
Nice catch, let's keep using 0 to disable memtable spill from now on.